db/seeds/users.rb

```diff
@@ -1 +1,2 @@
-User.create(email: 'somedude@gmail.com', user_name: 'somedude', password: 'password', password_confirmation: 'password')
+User.create(email: 'somedude@gmail.com', user_name: 'somedude', password: 'password', password_confirmation: 'password', admin: false)
+User.create(email: 'admin@admin.com', user_name: 'admin', password: 'admin', password_confirmation: 'admin', admin: true)
```

Comment:
Line is too long. [121/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.